pkg/scheduler/framework/plugins/dynamicresources/dynamicresources.go

```diff
@@ -399,6 +399,8 @@ func (pl *dynamicResources) EventsToRegister(_ context.Context) ([]framework.Clu
 		{Event: framework.ClusterEvent{Resource: framework.Node, ActionType: nodeActionType}},
 		// Allocation is tracked in ResourceClaims, so any changes may make the pods schedulable.
 		{Event: framework.ClusterEvent{Resource: framework.ResourceClaim, ActionType: framework.Add | framework.Update}, QueueingHintFn: pl.isSchedulableAfterClaimChange},
+		// Adding the ResourceClaim name to the pod status makes pods waiting for their ResourceClaim schedulable.
+		{Event: framework.ClusterEvent{Resource: framework.Pod, ActionType: framework.Update}, QueueingHintFn: pl.isSchedulableAfterPodChange},
```

Comment:
In order to reduce the number of subscribing events, we avoid using `Pod/Update` and instead use [specific events `Pod/UpdatePodXXXX`](https://github.com/kubernetes/kubernetes/blob/master/pkg/scheduler/framework/types.go#L69-L76).

So, I think you need to create a new one like `Pod/UpdatePodResourceClaims`.

Reply:
I was going to ask whether adding a specific event is rather overkill, given that there is only one plugin that cares about it, and it is alpha.
In other words, we could hide the complexity inside the hint, as opposed to make it part of the framework.

But, it's all about prons and cons. I'll leave the decision to @sanposhiho

Reply:
IIRC, the idea of fine-grained events is more important when there are no scheduling hints. But DRA is as close to graduating to beta as scheduling hints are, so maybe it's worth not adding extra complexity in the framework.

Reply:
Oh right, having fine grained events is helpful to reduce the number of stored inflight events, which affects scheduling throughput even if DRA is not used in the cluster. That sounds worse for the average user. Ok, let's keep the fine-grained event :)

Reply:
Yes, I think that was the reason why @sanposhiho suggested it.

If DRA is disabled, the new event isn't getting generated. But if it's enabled (not on by default while beta!) and merely not used (no ResourceClaims), then it will add some overhead.